src/ol_infrastructure/applications/jupyterhub/jupyterhub_policy.hcl

```diff
@@ -9,6 +9,10 @@ path "postgres-jupyterhub/creds/app" {
   capabilities = ["read"]
 }
 
+path "postgres-jupyterhub_authoring/creds/app" {
```

Comment:
Could easily split these out if we'd like. No reason to keep em all in one spot.